spec/features/visitors/password_recovery_spec.rb

```diff
@@ -166,8 +166,11 @@ def reset_password_and_sign_back_in
     before do
       sign_up_with_and_set_password_for('email@example.com')
       fill_in 'Mobile', with: '5555555555'
+      expect(Users::PhoneConfirmationController).
+        to receive(:generate_confirmation_code).and_return('1234')
       click_button 'Submit'
-      fill_in 'code', with: User.last.direct_otp
+
+      fill_in 'code', with: '1234'
```

Comment:
We can actually remove this entire test. It's already duplicated on line 25. The reason we had this test in save-ferris is because we had additional account creation steps after setting up 2FA, but in our case we don't.